test/e2e/test_manufacturer.py

```diff
@@ -194,6 +194,18 @@ def test_partial_address_update(test_client):
     }
 
 
+def test_update_captialistion_of_name(test_client):
```

Comment:
```suggestion
def test_update_capitalisation_of_name(test_client):
```

For #90 - The tests in this module should be checked and renamed if necessary. The update tests should be renamed so they start with `test_partial_update..` to keep them consistent with the e2e tests in the other modules.

Reply:
There are a few others that haven't been renamed.

Reply:
I was suggesting that they get addressed in #90.

Reply:
Oh right yes I see sorry.